src/content/docs/en/guides/integrations-guide/netlify.mdx

```diff
@@ -144,7 +144,7 @@ export default defineConfig({
 });
 ```
 
-Configuring `edgeMiddleware: true` will deploy your middleware as an Edge Function, and run it on all routes - including pre-rendered pages. However, locals specified in the middleware won't be available to any pre-rendered pages, because they've already been fully-rendered at build-time.
+When `edgeMiddleware` is enabled, an edge function will execute your middleware code for all requests for static assets, prerendered pages, and on-demand rendered pages. For on-demand rendered pages, the locals are serialized using JSON and sent in a header for the serverless function, which performs the rendering. As a security measure, the serverless function will refuse to serve requests with a `403 Forbidden` response unless they come from the generated edge function.
```

Comment:
```suggestion
When `edgeMiddleware` is enabled, an edge function will execute your middleware code for all requests including static assets, prerendered pages, and on-demand rendered pages.

For on-demand rendered pages, the locals are serialized using JSON and sent in a header for the serverless function, which performs the rendering. As a security measure, the serverless function will refuse to serve requests with a `403 Forbidden` response unless they come from the generated edge function.
```
- two paragraphs for readability

- slight tweak that DOES change the nuance (if correct, I like this better. If not correct, pls ignore): 

My version emphasizes the ALL-ness of it. If the main point is to emphasize that even your prerendered/static stuff is executed by the function, like everything, even non-SSR stuff, this slight change does that more explicitly. Your choice! Obviously do not change if the above *was* an exhaustive list of "these are the only things, out of many more possible things". But, if it's everything, I kind of like hitting that a little harder, because the sentence on its own doesn't tell you which is the case. Your call!

Reply:
It's accurate and much better to read, thanks!